docs/upgrade.html

```diff
@@ -29,6 +29,8 @@ <h5><a id="upgrade_240_notable" href="#upgrade_240_notable">Notable changes in 2
         avoid potential misuse. The constructor <code>TopicAuthorizationException(String)</code> which was previously used for a single
         unauthorized topic was changed similarly.
     </li>
+    <li>The internal <code>PartitionAssignor</code> interface has been deprecated and replaced with a new <code>ConsumerPartitionAssignor</code> in the public API. Users
+        implementing a custom PartitionAssignor should migrate to the new interface as soon as possible.</li>
```

Comment:
Should we also mention the newly added / changed functions as well?

Reply:
Do you mean list every method that changed between the interfaces? Seems cleaner to just direct people to the new interface rather than bloat the upgrade guide. Hopefully it will be obvious what/where they need to make changes?

Reply:
No need to list all functions, just mention that there are some minor signature changes, i.e. it is not a pure package.class renaming.

Reply:
Oh, ok. I'll tack this on to the Pt. 4 PR